### DISCUSSION TITLE
Is it possible to overlay threejs animations within model-viewer AR?

### DISCUSSION THREAD

**Discussion by EdwardDeaver** [google/model-viewer#3589] [Q&A]
Hello, 

I'm very excited to use this library. Is it possible to combine threejs animations within model viewer AR view? 

For example could I have a model of a trophy and have threejs particles flying around it in AR on my iphone? 

Best regards, 
Edward

**Comment by elalish** [comment]
Yes, you just need to export the animations inside of your GLB. Then you can use our animation API: https://modelviewer.dev/examples/animation/index.html
